newIDE/app/src/AssetStore/ExtensionStore/index.js

```diff
@@ -94,14 +101,30 @@ export const ExtensionStore = ({
         {windowWidth => (
           <ColumnStackLayout expand noMargin useFullHeight>
             <ColumnStackLayout>
-              <SearchBar
-                value={searchText}
-                onChange={setSearchText}
-                onRequestSearch={() => {}}
-                tagsHandler={tagsHandler}
-                tags={filters && filters.allTags}
-                placeholder={t`Search extensions`}
-              />
+              <ResponsiveLineStackLayout noMargin>
+                <SelectField
+                  value={chosenCategory}
+                  onChange={(e, i, value: string) => {
+                    setChosenCategory(value);
+                  }}
+                  disableUnderline
+                >
+                  <SelectOption value="" primaryText={t`All categories`} />
+                  {allCategories.map(category => (
+                    <SelectOption value={category} primaryText={category} />
```

Comment:
You should have a warning in the console stating that each `SelectOption` should have a unique key

Reply:
Setting `key={category}` should do the job